### PR TITLE
[ Xedra Evolved ] Vampire guide and mention urban fantasy in modinfo

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
None 

#### Purpose of change
Adds the vampirism starter guide to the markdown documents shipped with Xedra Evolved and opens with the term Urban Fantasy.

#### Describe the solution
Give people more options to read important things before clicking buttons.

#### Describe alternatives you've considered
Hiding a mininuke on all XE starting characters that can only be deactivated by inputting a password that is generated as a random word from the readme.

#### Testing
MD

### CODE REVIEW

**Comment by Rokharn** [comment]
I like alternative solution with mininuke, sure) But how about just put a book with info from readme into their inventories as one of starter item? As well as adds this book at the loot table (like 'vampirism for dummies' at vampire corps).

**Comment by Maleclypse** [comment]
> I like alternative solution with mininuke, sure) But how about just put a book with info from readme into their inventories as one of starter item? Also possible to add this book to the loot table (like 'vampirism for dummies' at vampire corps or so on).
> 
> Imho, ingame ways to learn things is always better for average player.

I appreciate that but I don't like to break the 4th wall or make things that the player could enjoy discovering on their own something that is handed to them.  

There are people in the world who want to discover a game blind and there are people who are unwilling to make any choice until they've found what the results will be.  I make games for the people who want to discover and I'm willing to make out of game concessions to people who want to know the future.

**Comment by Rokharn** [comment]
> discovering on their own

But to discovering process people still needs at least some more or less clear hints. For example, I do not think it's normal if so many players just do not understand how to eat blood (and even moment of actually needs it) and ask it time to time. So trivial, but still. And readme is a some kind of cheating for players, so some can prefer die by starvation with no clear understanding and next time just debug this illness,

File: data/mods/Xedra_Evolved/modinfo.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
...
+    "description": "An urban fantasy content mod for Agent Mulder in the apocalypse.  Creates a multi decade timeline of dimensional spillage prior to the cataclysm.  This spillage was characterized by thin places appearing where people could step through into another dimension but things from those dimensions could wander into our world as well.  Xedra proliferated into an alphabet soup of agencies dedicated to simultaneously stopping, controlling, and utilizing these thin places with varying degrees of success.  Contains monsters of various types, new professions and locations, new uses for existing skills plus the new deduction skill, new magic systems, and content of every type that a multidimensional Cataclysm has to offer.  This is a heavily science fantasy mod where you are as likely to encounter supernatural monsters, as you are to encounter UFOs and Sasquatch and Chupacabra.  Please examine the readme.md file for further details.",
```
Comment (MNG-cataclysm) [inline data/mods/Xedra_Evolved/modinfo.json:8]:
```suggestion
    "description": "An urban fantasy content mod for Agent Mulder in the apocalypse.  Creates a multi decade timeline of dimensional spillage prior to the Cataclysm.  This spillage was characterized by thin places appearing where people could step through into another dimension, but things from those dimensions could wander into our world as well. XEDRA proliferated into an alphabet soup of agencies dedicated to simultaneously stopping, controlling, and utilizing these thin places with varying degrees of success.  Contains monsters of various types, new professions and locations, new uses for existing skills,the new deduction skill, new magic systems, and content of every type that a multidimensional Cataclysm has to offer.  This is a heavily science-fantasy mod where you are as likely to encounter supernatural monsters as you are UFOs, Sasquatch, and Chupacabra.  Please examine the readme.md file for further details.",
```
Proofreading.

**Review by MNG-cataclysm — CHANGES_REQUESTED** [review]

File: data/mods/Xedra_Evolved/modinfo.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
     "name": "Xedra Evolved",
     "authors": [ "Maleclypse" ],
     "maintainers": [ "Maleclypse" ],
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/modinfo.json:7]:
```suggestion
    "maintainers": [ "Maleclypse", "Guardiandll" ],
```

File: data/mods/Xedra_Evolved/modinfo.json [hunk 3]
```diff
@@ -5,7 +5,7 @@
...
+    "description": "An urban fantasy content mod for Agent Mulder in the apocalypse.  Creates a multi decade timeline of dimensional spillage prior to the Cataclysm.  This spillage was characterized by thin places appearing where people could step through into another dimension, but things from those dimensions could wander into our world as well. XEDRA proliferated into an alphabet soup of agencies dedicated to simultaneously stopping, controlling, and utilizing these thin places with varying degrees of success.  Contains monsters of various types, new professions and locations, new uses for existing skills,the new deduction skill, new magic systems, and content of every type that a multidimensional Cataclysm has to offer.  This is a heavily science-fantasy mod where you are as likely to encounter supernatural monsters as you are UFOs, Sasquatch, and Chupacabra.  Please examine the readme.md file for further details.",
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/modinfo.json:8]:
```suggestion
    "description": "An urban fantasy content mod for Agent Mulder in the apocalypse.  Creates a multi decade timeline of dimensional spillage prior to the Cataclysm.  This spillage was characterized by thin places appearing where people could step through into another dimension, but things from those dimensions could wander into our world as well. 
 XEDRA proliferated into an alphabet soup of agencies dedicated to simultaneously stopping, controlling, and utilizing these thin places with varying degrees of success.  Contains monsters of various types, new professions and locations, new uses for existing skills,the new deduction skill, new magic systems, and content of every type that a multidimensional Cataclysm has to offer.  This is a heavily science-fantasy mod where you are as likely to encounter supernatural monsters as you are UFOs, Sasquatch, and Chupacabra.  Please examine the readme.md file for further details.",
```

File: data/mods/Xedra_Evolved/modinfo.json [hunk 4]
```diff
@@ -4,8 +4,9 @@
...
+    "maintainers": [ "Maleclypse", "Guardiandll" ],
+    "description": "An urban fantasy content mod for Agent Mulder in the apocalypse.  Creates a multi decade timeline of dimensional spillage prior to the Cataclysm.  This spillage was characterized by thin places appearing where people could step through into another dimension, but things from those dimensions could wander into our world as well. 
+ XEDRA proliferated into an alphabet soup of agencies dedicated to simultaneously stopping, controlling, and utilizing these thin places with varying degrees of success.  Contains monsters of various types, new professions and locations, new uses for existing skills,the new deduction skill, new magic systems, and content of every type that a multidimensional Cataclysm has to offer.  This is a heavily science-fantasy mod where you are as likely to encounter supernatural monsters as you are UFOs, Sasquatch, and Chupacabra.  Please examine the readme.md file for further details.",
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/modinfo.json:9]:
```suggestion
```

File: data/mods/Xedra_Evolved/modinfo.json [hunk 5]
```diff
@@ -4,8 +4,9 @@
...
+    "maintainers": [ "Maleclypse", "Guardiandll" ],
+    "description": "An urban fantasy content mod for Agent Mulder in the apocalypse.  Creates a multi decade timeline of dimensional spillage prior to the Cataclysm.  This spillage was characterized by thin places appearing where people could step through into another dimension, but things from those dimensions could wander into our world as well.
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/modinfo.json:8]:
```suggestion
    "description": "An urban fantasy content mod for Agent Mulder in the apocalypse.  Creates a multi decade timeline of dimensional spillage prior to the Cataclysm.  This spillage was characterized by thin places appearing where people could step through into another dimension, but things from those dimensions could wander into our world as well. 
 XEDRA proliferated into an alphabet soup of agencies dedicated to simultaneously stopping, controlling, and utilizing these thin places with varying degrees of success.  Contains monsters of various types, new professions and locations, new uses for existing skills,the new deduction skill, new magic systems, and content of every type that a multidimensional Cataclysm has to offer.  This is a heavily science-fantasy mod where you are as likely to encounter supernatural monsters as you are UFOs, Sasquatch, and Chupacabra.  Please examine the readme.md file for further details.",
```

**Pull request merged**